Makefile

```diff
@@ -41,14 +42,20 @@ compile-locale:
 	./bin/find-good-catalogs src/sentry/locale/catalogs.json
 	cd src/sentry && sentry django compilemessages
 
-locale: merge-locale-catalogs compile-locale
-
-sync-transifex: merge-locale-catalogs
```

Comment:
split up push and pull because it's not normal you would do them sequentially like that.